docs/book/src/tasks/installation.md

```diff
@@ -18,14 +18,15 @@ Cluster API requires an existing kubernetes cluster accessible via kubectl, choo
 
 <h1>Warning</h1>
 
-**Minimum supported version**: v0.6.x
+**Minimum supported version of kind**: v0.6.x
```

Comment:
```suggestion
**Minimum KIND supported version**: v0.6.x
```

Reply:
kind should be lowercased: https://kind.sigs.k8s.io/